apis.h

```diff
@@ -19,6 +19,8 @@ int api_send_transfer(const iota_client_service_t* const service,
                       const char* const obj, char* json_result);
 int api_get_transaction_msg(const iota_client_service_t* const service,
                             const char* const obj, char* json_result);
+int api_find_transactions_by_tag(const iota_client_service_t* const service,
+                                 const char* const obj, char* json_result);
```

Comment:
Parameter `json_result` has to be `char**` or original value won't change.

Reply:
Right! I'll modify them all